pkgs/applications/gis/gmt/default.nix

```diff
@@ -1,59 +1,98 @@
-{ lib, stdenv, fetchurl, cmake, curl, Accelerate, CoreGraphics, CoreVideo
-, fftwSinglePrec, netcdf, pcre, gdal, blas, lapack, glibc, ghostscript, dcw-gmt
-, gshhg-gmt }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  curl,
+  Accelerate,
+  CoreGraphics,
+  CoreVideo,
+  fftwSinglePrec,
+  netcdf,
+  libxml2,
+  pcre,
+  gdal,
+  blas,
+  lapack,
+  glibc,
+  ghostscript,
+  dcw-gmt,
+  gshhg-gmt,
+}:
 
-/* The onus is on the user to also install:
-    - ffmpeg for webm or mp4 output
-    - graphicsmagick for gif output
+/*
+  The onus is on the user to also install:
+   - ffmpeg for webm or mp4 output
+   - graphicsmagick for gif output
 */
 
-stdenv.mkDerivation rec {
+let
+  # Certainly not an ideal situation, See:
+  # https://github.com/NixOS/nixpkgs/pull/340707#issuecomment-2361894717
+  netcdf' = netcdf.override {
```

Comment:
I think you don't need that now, because you've already overriden it in all-packages.

Reply:
> I think you don't need that now, because you've already overriden it in all-packages.

Thanks for catching that. I meant to only override things inside the expression.